snap/snapcraft.yaml

```diff
@@ -27,23 +27,20 @@ parts:
       - config.yaml
 
 plugs:
-  # For reading config file from user home directory as root
-  home:
-    read: all
```

Comment:
Why dropping this?
It was added to make it possible to run the tool as root with configurations placed in a normal user directory; see https://github.com/canonical/rt-conf/pull/49